### PR TITLE
add clint, args, and pkginfo packages

### PR DESCRIPTION
**Pull request by janschulz**
These packages are needed for twine, which I want to package as well...

### CODE REVIEW

File: recipes/args/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: args
+  version: {{ version }}
+
+source:
+  fn: args-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/e5/1c/b701b3f4bd8d3667df8342f311b3efaeab86078a840fb826bd204118cc6b/args-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/args/meta.yaml]: Switch to `https://pypi.io/packages/source/a/args/args-0.1.0.tar.gz`

**Comment by jakirkham** [comment]
Could we please try using pypi.io for the URLs? One example above.

File: recipes/args/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - args
+
+about:
+  home: https://github.com/kennethreitz/args
+  license: BSD License
```
Comment (jakirkham) [inline recipes/args/meta.yaml]: Please specify number of clauses (e.g. BSD 3-Clause).

File: recipes/pkginfo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - pkginfo
+
+about:
+  home: https://pypi.python.org/pypi/pkginfo/
+  license: Python
```
Comment (jakirkham) [inline recipes/pkginfo/meta.yaml]: Maybe PSF would be more clear.

**Comment by jakirkham** [comment]
Also terminal newlines are nice. 😄

File: recipes/args/meta.yaml (hunk 2 above)
Comment (janschulz) [inline recipes/args/meta.yaml]: I do hope that this is that: https://github.com/kennethreitz/args/blob/master/LICENSE

File: recipes/args/meta.yaml (hunk 1 above)
Comment (janschulz) [inline recipes/args/meta.yaml]: done

File: recipes/pkginfo/meta.yaml (hunk 3 above)
Comment (janschulz) [inline recipes/pkginfo/meta.yaml]: done... also filed a bug in conda-build about this :-)

**Comment by janschulz** [comment]
Added newlines in the end, shortened the URLs and changed the license infos to be more in line with best practises

**Comment by janschulz** [comment]
Apart from the py 3.5 problems on windows (#666) it's passing :-) I think it's probably best not to merge this before a fix for that is released...

**Comment by ericdill** [comment]
@janschulz if you retrigger the CI's you should get a green checkmark here

**Comment by ocefpaf** [comment]
@janschulz some packages `clint` depends on `args`, so you might need to re-start the CIs in the feedstock after `args` is built.

**Pull request merged**